app/frontend/components/router.js

```diff
@@ -44,6 +44,4 @@ const TopLevelRouter = connect((state) => ({
   return content
 })
 
-module.exports = {
-  TopLevelRouter,
-}
```

Comment:
Let's keep it consistent and export this as default, so that places it's imported in don't have to specify extra {}

Reply:
I'll do this one as it clearly exports a component but I am reluctant to do others as there it is not so clear what is being exported.